src/components/NftPreview/NftPreviewLabel.tsx

```diff
@@ -27,14 +32,43 @@ export const StyledNftPreviewLabel = styled.div`
   padding: 8px;
   z-index: 10;
   justify-content: end;
-  height: 100%;
+  // height: 100%;
   // this helps position the label correctly in Safari
   // Safari differs from Chrome in how it renders height: 100% on position: absolute elements
   min-height: 56px;
 `;
 
-const StyledBodyRegular = styled(BodyRegular)`
+const StyledBodyRegular = styled(BodyRegular)<{ lines: number }>`
+  word-wrap: break-word;
+  word-break: break-all;
+
   margin: 0;
+  text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.7);
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  line-clamp: ${({ lines }) => lines};
+  -webkit-line-clamp: ${({ lines }) => lines};
+  overflow: hidden;
```

Comment:
do you know why we're using `overflow: hidden`? looks like this only gets applied on mobile, but it causes a weird box-in effect on the text shadow:
![Screen Shot 2022-02-22 at 12 43 44 AM](https://user-images.githubusercontent.com/12162433/155070544-2cad6961-a904-48b1-a3e1-b47b6c6bc086.png)

removing it fixes the issue. looks like desktop doesn't get `overflow: hidden` applied
![Screen Shot 2022-02-22 at 12 44 22 AM](https://user-images.githubusercontent.com/12162433/155070553-3fa23006-e23b-4d2e-b83d-62f9bbe9e7b3.png)